releasenotes/notes/34246.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 34246
+releaseNotes:
+  - |
+    **Added** support for NodePort eastwest gateway when
```

Comment:
Maybe expand this a bit to add more context for someone unfamiliar with the problem?

Reply:
Done.